assets/js/hooks/useInView.test.js

```diff
@@ -0,0 +1,144 @@
+/**
+ * `useInView` hook tests.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { actHook as act, renderHook } from '../../../tests/js/test-utils';
+import { CORE_UI } from '../googlesitekit/datastore/ui/constants';
+import { useInView } from './useInView';
+
+describe( 'useInView', () => {
+	it( 'should return true when a parent <InViewProvider /> has a `true` value', () => {
+		const { result } = renderHook( () => useInView(), { inView: true } );
+
+		expect( result.current ).toEqual( true );
```

Comment:
@tofumatt, could you, please, rename all `.toEqual( true )` and `.toEqual( false )` in this file to be `.toBe( true )` and `.toBe( false )`? The `.toEqual( ... )` function is intended to compare objects, not scalar values.